src/aui/dockart.cpp

```diff
@@ -337,11 +337,11 @@ void wxAuiDefaultDockArt::UpdateColoursFromSystem()
     //wxColor darker4Colour = baseColour.ChangeLightness(50);
     wxColor darker5Colour = baseColour.ChangeLightness(40);
 
-    m_activeCaptionColour = wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHT);
-    m_activeCaptionGradientColour = wxAuiLightContrastColour(wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHT));
-    m_activeCaptionTextColour = wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHTTEXT);
-    m_inactiveCaptionColour = darker1Colour;
-    m_inactiveCaptionGradientColour = baseColour.ChangeLightness(97);
+    m_activeCaptionColour = wxSystemSettings::GetColour(wxSYS_COLOUR_ACTIVECAPTION);
```

Comment:
It does seem to make sense to use `ACTIVECAPTION` for, well, active caption, but we need to change `src/osx/cocoa/settings.mm` to map it to something different than `windowFrameColor` which is almost certainly not appropriate.

Reply:
Why is it not appropriate? Is the window frame color different than the title bar color?